libraries/habitat_shared.rb

```diff
@@ -27,7 +27,7 @@ def hab(*command)
       if Gem::Requirement.new('>= 14.3.20').satisfied_by?(Gem::Version.new(Chef::VERSION))
         shell_out!(hab_cmd)
       else
-        shell_out_compact!(hab_cmd) # cookstyle: disable ChefDeprecations/DeprecatedShelloutMethods
```

Comment:
oops, looks like when I ran cookstyle locally it fixed this automatically, I'll revert